### ISSUE TITLE
Incorrect behavior for guessing suite_num

### ISSUE THREAD

**Issue by artemijan** [pnpnpn/street-address#15]
Hello,
there is a case for this address '268 S Beverly Dr. #9876'
This address is parsed and suite_num is None.
I've found the [reason](https://github.com/pnpnpn/street-address/blob/master/streetaddress/streetaddress.py#L93)
Take a look:
`
elif len(word_lw) > 0 and word_lw[0] == '#' and res['suite_num'] is not None
`
should be without "not" statement
`
elif len(word_lw) > 0 and word_lw[0] == '#' and res['suite_num'] is None
`
What do you think?

**Comment by pnpnpn** [comment]
Yeah, you are right. It's better to test that both suite type and name are None.

Can you create a pull request? Thanks.

**Comment by artemijan** [comment]
Yes, I will try to do it today
